src/shared/components/common/wine-price-slider/index.tsx

```
@@ -0,0 +1,79 @@
+'use client';
+/**
+ * @author: Hyun
+ * @since: 2025-06-14
+ * @description: 와인 가격 슬라이더 컴포넌트
+ */
+import DualRangeSlider from '@/shared/components/ui/dual-range-slider';
+import { useState } from 'react';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)